src/pages/Content/index.js

```diff
@@ -1,15 +1,147 @@
+
+var recordedEvents = []; // store events
+var clipboard = []; // store user copied text
+
+// ---------------------------------------------------
+// ATTACH GLOBAL EVENT LISTENER
+// ---------------------------------------------------
+
+function attachGlobalEventListeners() {
+  /*
+  a function that listens for 4 events:
+    - click (left mouse)
+    - contextmenu (right mouse)
+    - keydown events
+        >> handles multi-button presses (key + Ctrl + Alt + Shift)
+        >> handles copiying text to global clipboard
+    - URL change
+  */
+
+  // left click listener
+  // ---------------------------------------------------
+  document.body.addEventListener('click', function (e) {
+    recordedEvents.push({
+      type: 'click',
+      element: e.target,
```

Comment:
We don't need the `element` but the `element`s selector, try using something like:
https://github.com/antonmedv/finder

Reply:
I see.